Azure-Samples/Cognitive-Services-Voice-Assistant
# Fix for App losing scope and Delete Profile button

## Purpose
* Previously the app was losing scope when deactivated (switching to another app) - text fields were reverting back to the original text from previous profile. This fix is to ensure that the values entered in the respective fields are not lost when application is deactivated. 
* Combo boxes with previous history of Subscription Key, Region, and Custom Commands App Id have been replaced with Text Boxes.
* Delete Profile button has been added to remove a profile and its associated values from app history. 

## Does this introduce a breaking change?
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
[x] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Create a new profile
* Copy and paste the values being tested - Text fields should keep the values pasted
* Connect to Bot
* Create a new profile, save profile, open settings and delete the created profile.
  * Upon clicking delete, all populated fields along with the profile name will become empty and deleted profile will be removed from the Profile History list.
